### DISCUSSION TITLE
Resources.LoadAll 동작이 이상합니다...

### DISCUSSION THREAD

**Discussion by MChis90** [naverz/zepeto-studio-kor#1265] [5. Q&A]
[환경]
OS : Winodw10
윈도우 버전:
![image](https://user-images.githubusercontent.com/83467461/188611254-b54d7104-0412-436f-bc9d-8be09e87e9a2.png)

유니티 버전 : 2020.3.9f1
제페토 버전 :
![image](https://user-images.githubusercontent.com/83467461/188610587-1bb55f7d-7bd9-4226-84f0-697d7f723f00.png)

---------------------------------------------------------------------------------------------------------------------------------------
![image](https://user-images.githubusercontent.com/83467461/188609792-7074c77d-9908-4e2e-8ee5-e7c11f66b7f7.png)
현재 Resources 디렉토리 입니다

코드:
![image](https://user-images.githubusercontent.com/83467461/188611780-2cfe9294-3794-462e-8de3-188c89ded1c2.png)

Resources 디렉토리에 하위 디렉토리를 만들고 타겟이 되는 디렉토리 경로를 
Resources.LoadAll 함수를 이용하여 타겟 디렉토리 내부의 프리팹과 스프라이트, json텍스트 파일들을 로드 하려고 합니다

PC에서 프로젝트 실행 시 정상적으로 Resources 하위 디렉토리의 오브젝트들을 읽어 오는 것이 확인 되었습니다.

Json파일 5개 Load:
![image](https://user-images.githubusercontent.com/83467461/188612397-1bd654ee-ea8f-426e-a341-e959079b4d02.png)

하지만 QR로 빌드 시 모바일 환경에서 타겟 디렉토리를 대신 Resources 폴더의 모든 오브젝트들을 로드 합니다.

모바일에서 확인 했을 때:
![image](https://user-images.githubusercontent.com/83467461/188613882-ce24295a-232e-493d-a8fe-5dd9b6168b81.png)

Resources.LoadAll 함수를 원래 쓸 수 없는 건가요?

아니면 제가 뭔가 잘못 사용하고 있는 걸까요?

**Comment by YSHFM** [comment]
아 저도 동일하게 저런 이슈가 있었습니다 이슈리포트에 작성하려했지만 시간이 없었던.....
